zerver/lib/actions.py

```diff
@@ -2826,8 +2827,12 @@ def bulk_add_subscriptions(streams: Iterable[Stream],
         else:
             color = pick_color(user_profile, subs_by_user[user_profile.id])
 
+        role = Subscription.ROLE_MEMBER
+        if stream_admin_map is not None:
+            if user_profile.id in stream_admin_map and stream in stream_admin_map[user_profile.id]:
+                role = Subscription.ROLE_STREAM_ADMINISTRATOR
```

Comment:
I think I'd prefer to have the commit that makes it possible to create stream administrators sit later in the branch, after we've polished the changes that implement the basic logic for what they can do.  Can you move this commit a few steps later?  

Does this change result in `populate_db` creating any streams with a stream administrator?

Reply:
I have moved the commit. 
Currently I have not changed the code in populate_db to make stream administrator, will do this.